<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manualTestDialog</class>
 <widget class="QDialog" name="manualTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>手动测试界面</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>解析设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>协议选择：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>BE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中间件</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="inputDataEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="clearInputBtn">
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="parseBtn">
          <property name="text">
           <string>解析</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>解析结果</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,1">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>结果</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QTableWidget" name="resultTableWidget">
           <column>
            <property name="text">
             <string>表号</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>编码</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>名称</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>源码</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>工程值</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>含义</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>信息提示</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="clearBtn">
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
